### PR TITLE
Added immersive compat rectangle background

### PR DESCRIPTION
**Pull request by yoksnod**
For samsung 8 or samsung 9 or another device FullScreenPromptBackground drawn above navigation bar when user transit to immersive mode.
E.g here:
before using new Background class - https://s.mail.ru/Fs5b/8NpwtvYtg
after using new Background class - https://cloud.mail.ru/public/KtQi/huG5YMyCo
My code sample https://s.mail.ru/4Fdb/uhhv6Wixd

Please, take a look my pull request. Thank you

### CODE REVIEW

File: library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/ImmersiveModeCompatPromptBackground.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+package uk.co.samuelwall.materialtaptargetprompt.extras.backgrounds;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.WindowManager;
+
+public class ImmersiveModeCompatPromptBackground extends FullscreenPromptBackground
```
Comment (sjwall) [inline library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/ImmersiveModeCompatPromptBackground.java]: Could you add JavaDoc here describing the purpose for the class please

**Comment by sjwall** [comment]
Thanks for the PR! It looks good. Just one comment to add JavaDoc to the new class to describe its purpose. I'll merge it in once it has been added

File: library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/ImmersiveModeCompatPromptBackground.java (hunk 1 above)
Comment (yoksnod) [reply library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/ImmersiveModeCompatPromptBackground.java]: Yes, of course. E.g. as here https://s.mail.ru/4DW8/XKexBcaBx ?

**Comment by yoksnod** [comment]
@sjwall Done. Please, take a look

**Comment by sjwall** [comment]
Sorry for the delay, this will be in the next release

**Comment by yoksnod** [comment]
Thanks, great!

**Pull request merged**